### PR TITLE
Added script to manage default subreddits

### PR DESCRIPTION
**Pull request by rhysyngsun**
This adds a script to manage the default subreddits

Example usages are in the script itself

### CODE REVIEW

File: scripts/default_subreddits.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Usage:
+#    To list default subreddits:
+#       sudo reddit-run scripts/default_subreddits.py -c 'list_default_subreddits()'
+#    To set default subreddits:
+#       sudo reddit-run scripts/default_subreddits.py -c 'set_default_subreddits("one", "two", "three")'
+#    To add a default subreddit:
+#       sudo reddit-run scripts/default_subreddits.py -c 'add_default_subreddits("four")'
```
Comment (mbertrand) [inline scripts/default_subreddits.py]: typo, should be `'add_default_subreddit("four")'`

File: scripts/default_subreddits.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+#    To remove a default subreddit:
+#       sudo reddit-run scripts/default_subreddits.py -c 'remove_default_subreddits("five")'
```
Comment (mbertrand) [inline scripts/default_subreddits.py]: should be `remove_default_subreddit("five")`

**Review by mbertrand — CHANGES_REQUESTED** [review]
Works great, just a couple small typos in the Usage section.

**Comment by rhysyngsun** [comment]
@mbertrand all set

**Review by mbertrand — APPROVED** [review]

**Pull request merged**